calendar/manifest.json

```diff
@@ -49,30 +49,30 @@
 
   "experiment_apis": {
     "calendar_calendars": {
-      "schema": "api/schema/calendar-calendars.json",
+      "schema": "experiments/calendar/schema/calendar-calendars.json",
```

Comment:
Let's leave this file as is so the built in extension for this experiment works.